examples/profiling/README.md

```diff
@@ -0,0 +1,27 @@
+# Profiling Example
+
+> :warning: Profiling is still an experimental feature
+
+This example showcases enabling profiling for Splunk APM. There's no official support for profiling in OTel, so profiling requires working with some Splunk-specific components.
+By default, the example requires the OTel Collector to run with the OTLP receiver listening for logs on `localhost:4317`. To export profiling data to the APM Collector must be used with `splunk_hec` exporter. See [the example collector config](./collector-config.yml).
```

Comment:
"To export profiling data to the APM Collector must be used with `splunk_hec` exporter. " is confusing. 

What do we mean, exactly? Is there a comma missing, as in "To export profiling data to APM, the Collector must be used with `splunk_hec` exporter."?

Reply:
Ingest requires `splunk_hec` which can be set up in the collector, so contrary to tracing telemetry, which can be exported directly to the APM, profiling currently requires collector.

Reply:
How about: "To export profiling data to APM, you must set up splunk_hec in the Collector".

Reply:
Made the change and merged. Thanks.